README.md

```diff
@@ -1,36 +1,39 @@
+| **News** | ![![Twitter Follow](https://twitter.com/turingway)](https://img.shields.io/twitter/follow/turingway?style=social) [![Join our tinyletter mailing list](https://img.shields.io/badge/receive-our%20newsletter%20❤%EF%B8%8F-blueviolet.svg)](https://tinyletter.com/TuringWay)  ![![YouTube Channel Views](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)](https://img.shields.io/youtube/channel/views/UCPDxZv5BMzAw0mPobCbMNuA?style=social)  |
+| **Chat** | ![https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw](https://img.shields.io/badge/Chat-on%20Slack-ff69b4) ![GitHub issues](https://img.shields.io/github/issues/alan-turing-institute/the-turing-way) ![GitHub pull requests](https://img.shields.io/github/issues-pr/alan-turing-institute/the-turing-way) |
+| **Build** | [![Netlify build](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) [![Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
+| **Cite** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |
 
-**_This README.md file in also available in Dutch ([README-Dutch](README-translated/README-Dutch.md)), French ([README-French.md](README-translated/README-French.md)), German ([README-German.md](README-translated/README-German.md)), Indonesian ([README-Indonesian](README-translated/README-Indonesian.md)), Italian ([README-Italian](README-translated/README-Italian.md)), Korean ([README-Korean](README-translated/README-Korean.md)), Portuguese ([README-Portuguese](README-translated/README-Portuguese.md)), and Spanish ([README-Spanish](README-translated/README-Spanish.md)) (listed alphabetically)._**
+_The Turing Way_ is an open source project that aims to make data science and research reproducible, ethical and collaborative for everyone. 
```

Comment:
```suggestion
_The Turing Way_ is an open-source project that aims to make data science and research reproducible, ethical and collaborative for everyone. 
![The Turing Way project is a book, community, an open-source project and a culture of collaboration. This is illustrated in four types of illustrations, the first one showing the Turing Way book, the second showing how the community can grow, the fourth one showing two people collaborating on a pull request, the last one is showing a balance where reproducibility is valued more than the number of paper published .](book/website/figures/README_imgs/README_turingway.png)
```

Reply:
I think the slide which is widely used in the TW talks is very nice to add to the README. It describes the project beautifully!

Reply:
I'm not sure if there is a githubbish way that I can _suggest a change_ to an already suggested change so I'll write this out:

First, love it! I think this image is a nice way to welcome people and quickly introduce them to what is happening.

> This is illustrated in four types of illustrations

is a little difficult to read. May I suggest "This is illustrated/shown/depicted/visualised in four illustrations" instead?

There are 2 typo/issues 
- I think "fourth" should be "third"
- There is a space before the full stop at the end.

Reply:
Thank you @da5nsy, I updated it as you suggested!

Reply:
Glad my suggestions were useful :)
I included "illustrated/shown/depicted/visualised" as multiple suggested options and expected you to pick one and discard the others though! 😄

Reply:
This is hilarious 😂..